features/user-access-management/README.md

```diff
@@ -20,24 +20,33 @@ Go to the [Users section](https://imagekit.io/dashboard?redirectTo=users) in the
 
 ## User roles
 
-A user can have five different roles:
+A user can have six different roles:
 
 ### Account administrator
 
 User with this role has full access to your ImageKit.io account and can add other users with account administrator role. Best for people managing the complete account.
 
-### Developer&#x20;
+### Developer
 
-A user with a developer role can access everything except user management and billing. Your development team member should have this role.&#x20;
+A user with a developer role can access everything except user management and billing. Your development team member should have this role.
 
-### Media library full access&#x20;
+### Media library full access
 
 A user with this role has full access to the media library. This role is suitable for marketing team members who need to upload marketing assets, create new folders or tag media files etc.
 
 ### Media library view-only access
 
 A user with this role has view-only access to the media library. This user cannot upload media assets or create new folders. The user can only open and search the already upload files & folders.
 
+### Media library restricted access
+
+{% hint style="info" %}
+**Enterprise plan only**
+This user is only available in custom enterprise pricing plans.
+{% endhint %}
+
+A user with this role has restricted access to the media library. Based on their permission level, this user can only access and operate on files, folders, and media collections shared with them.
+
```

Comment:
Let's add a point that explains when initially a restricted ML user is added, they would not be able to create folders or upload files to the Media Library until an existing folder is shared with them with at least "Can contribute" access.

Reply:
This information is already given in folders and file upload pages under the access control and permissions section.

We also inform the users that they can only work on assets they have been shared. Writing that information here would be duplicate and a bit out of context.

Reply:
Under ACL section, we have described what the user cannot do until assets are shared. 

It would be useful for the admin to understand what they need to do to actively onboard a restricted user into contributing to the ML. Perhaps we can phrase it as quickstart steps, e.g.:

1. Add a restricted user from the User Management panel.
2. Share a folder with the newly created restricted user. Providing "Can contribute" or "Can manage" access to this folder will allow them to start uploading their own assets.

Reply:
Added more description for user's help

Reply:
Looks fine for now, though I was thinking of a small subsection on `access-control-and-permissions/README.md` page. It can be done later.